Taskfile.yml

```diff
@@ -313,7 +313,6 @@ tasks:
         --handler=sqlite
         --sqlite-url=file:tmp/sqlite/
         --test-records-dir=tmp/records
-        --test-disable-filter-pushdown
```

Comment:
Yes, removing this would result in not test `test-disable-filter-pushdown` flag being true for `sqlite`.

I think postgres is independent from this change, postgres integration test only case where `test-disable-filter-pushdown` is set false.

We do test `$eq` filter works correctly for both `sqlite` and `pg`.